config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presubmits.yaml

```diff
@@ -0,0 +1,15 @@
+presubmits:
+  kubernetes-sigs/cluster-api-provider-vsphere:
+  - name: pull-cluster-api-provider-vsphere-test
+    always_run: true
+    decorate: true
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/cluster-api:v20180720-d52d72975
```

Comment:
just use latest kubekins, aka `gcr.io/k8s-testimages/kubekins-e2e:v20181010-2672d9494-master`?

this cluster-api image seems stale.

Reply:
@krzyzacy got you, i actually following the GCP example, and don't know the state of those container images, thanks for review it, i will switch.